### ISSUE TITLE
Ruby 3.0 breaks web-console

### ISSUE THREAD

**Issue by perryventas** [rails/activerecord-session_store#173]
Using the `activerecord-session_store` gem on a Rails 6.1 application with latest Ruby 3.0 seems to break the web-console on development environment and instead renders the public/500.html error page.

How to reproduce:
1. Create a fresh Rails 6.1 application with Ruby 3.0, start webserver and make sure web-console works (e.g. triggering a 404 by entering an unknown URL)
<img width="347" alt="Screenshot 2021-03-07 at 16 28 48" src="https://user-images.githubusercontent.com/617255/110245337-ef1b5300-7f62-11eb-87ba-887e46aaf37c.png">
2. Add the `activerecord-session_store` to your Gemfile, run bundle install, restart the rails server and try again to trigger a 404, which will result in displaying the default public/500.html error page:

<img width="666" alt="Screenshot 2021-03-07 at 16 38 43" src="https://user-images.githubusercontent.com/617255/110245507-b62fae00-7f63-11eb-87cb-b09c4f1fae93.png">

The console does not show anything suspicious:
```
Started GET "/asdf" for 127.0.0.1 at 2021-03-07 16:38:37 +0100
   (3.4ms)  SELECT sqlite_version(*)

ActionController::RoutingError (No route matches [GET] "/asdf"):
```

Downgrading the Ruby version to 2.7.2 fixes the issue.

Thanks in advance

**Comment by denny** [comment]
There seem to be some recent issues in this repo relating to 6.1 as well.

Fwiw I'm using this gem in a Ruby 3.0, Rails 6.1 app myself ([ShinyCMS](/denny/ShinyCMS-ruby) and it seems fine, but it's not in production use.

**Comment by byroot** [comment]
2.0.0 was released and contains the Ruby 3.0 compatibility fix.
